admin/tests/integration/components/sessions/jury-comment_test.js

```diff
@@ -0,0 +1,32 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import moment from 'moment';
+
+module('Integration | Component | Sessions::JuryComment', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    // given
+    this.set('author', 'Vernon Sanders Law');
+    this.set('date', new Date('2021-06-21T14:30:21Z'));
+    this.set('comment', 'L\'expérience est un professeur cruel car elle vous fait passer l\'examen, avant de vous expliquer la leçon.');
```

Comment:
Depuis ember octane on peut maintenant direct faire ça qui est moins lourd syntaxiquement :
```suggestion
    this.author = 'Vernon Sanders Law';
    /*...*/
```
C'est juste que dans le code on trouve la syntaxe avec .set car à une époque on n'avait pas le choix